### ISSUE TITLE
Document the autoencoder architecture and changes

### ISSUE THREAD

**Issue by eliseoe** [maxweissenbacher/latentRL#9]
Now that we converged to the symlog CAE, @eliseoe  should document this.

**Comment by maxweissenbacher** [comment]
Hi Elise, in the process of setting up some code to try and train both agent and CAE at the same time - can you let me know some 'good' (not necessarily optimal but good enough) values for the latent dimension?

If I've interpreted your code correctly, then

batch size = 512
learning rate = 0.001
epochs = 1000

Is that correct? (for nu = 0.05)

**Comment by eliseoe** [comment]
![LatentRL_WandB_Screenshot_CAE_nu0_05](https://github.com/maxweissenbacher/latentRL/assets/49257902/5bdb1ba3-02cb-4993-8810-01168c640803)

@maxweissenbacher yes, this is correct! I attached a screenshot from wandb. I would recommend trying a latent size of 16 as the validation error is of order 10^(-3). For a smaller order of magnitude, you would have to try 36. 

I can also run a few smaller latent sizes, let me know!
